Makefile

```diff
@@ -0,0 +1,107 @@
+BACKEND?=docker
+CONCURRENCY?=1
+
+
+export LUET?=$(shell which luet)
+export ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+COMPRESSION?=zstd
+ISO_SPEC?=$(ROOT_DIR)/iso/sampleOS.yaml
+CLEAN?=false
+export TREE?=$(ROOT_DIR)/packages
+
+BUILD_ARGS?=--pull --no-spinner --only-target-package --live-output
+FLAVOR?=opensuse
+VALIDATE_OPTIONS?=-s
+DESTINATION?=$(ROOT_DIR)/build
+REPO_CACHE?=raccos/sampleos
+FINAL_REPO?=raccos/releases-sampleos
+
+PACKAGES?=$(shell yq r -j $(ISO_SPEC) 'packages.[*]' | jq -r '.[]' | sort -u)
+HAS_LUET := $(shell command -v luet 2> /dev/null)
+PUBLISH_ARGS?=
+ISO?=$(ROOT_DIR)/$(shell ls *.iso)
+
+export REPO_CACHE
+ifneq ($(strip $(REPO_CACHE)),)
+	BUILD_ARGS+=--image-repository $(REPO_CACHE)
```

Comment:
Just a note about the image reusal, luet 0.12.0 allows now to pull from different caches, see : https://github.com/mudler/luet/issues/185#issuecomment-815562222

Reply:
Yes I added the `--pull-repository` build flag in the Makefile :+1: